src/site/markdown/index.md

```diff
@@ -340,8 +386,53 @@ If the pixels are ints, that’s  an 8 GB file.  We can do
 ```
     
 The `reset()` method causes the internal stream to seek to the beginning of the data area.
-If that’s not possible it returns false. 
+If that’s not possible it returns false.
+
+#### Random Access
+It is possible to specify alternate random access by implementing the `RandomAccessFileIO` interface.  This is useful
+when a Fits file isn't necessarily on disk, but rather in object storage on the web, for example, or any other
+network access.
 
+```java
```

Comment:
I'm not sure we need to spell out in the README how to implement the interface, given especially that the example implementation is pretty much trivial (yet long).